reactor-netty-http/src/main/java/reactor/netty/http/client/WebsocketClientOperations.java

```diff
@@ -240,32 +240,22 @@ public NettyOutbound send(Publisher<? extends ByteBuf> dataStream) {
 	}
 
 	@SuppressWarnings("FutureReturnValueIgnored")
-	void sendCloseNow(@Nullable CloseWebSocketFrame frame) {
-		if (frame != null && !frame.isFinalFragment()) {
+	void sendCloseNow(CloseWebSocketFrame frame, WebSocketCloseStatus closeStatus) {
+		if (!frame.isFinalFragment()) {
```

Comment:
Really minor but `WebSocketCloseStatus` could be `@Nullable`, and if not provided, derived as previously so that way for the normal case `CloseWebSocketFrame` could contain the status but for the 1006 case it is provided separately.

Reply:
done